packages/authentication/modules/user/useUser/index.ts

```diff
@@ -1,16 +1,28 @@
-import { COOKIE_NAME } from '@baseapp-frontend/utils'
+import { ACCESS_COOKIE_NAME, decodeJWT } from '@baseapp-frontend/utils'
 
 import { useQuery, useQueryClient } from '@tanstack/react-query'
 import Cookies from 'js-cookie'
 
 import UserApi, { USER_API_KEY } from '../../../services/user'
 import { IUser } from '../../../types/user'
-import { IUseUser } from './types'
+import { IUseSimpleTokenUser, IUseUser } from './types'
 
-const useUser = <TUser extends Partial<IUser>>({
+const useUser = ({ cookieName = ACCESS_COOKIE_NAME }: IUseUser = {}): IUser | null => {
+  const token = Cookies.get(cookieName)
+  if (token) {
+    try {
+      return decodeJWT<IUser>(token)
+    } catch (error) {
+      return null
+    }
+  }
+  return null
+}
+
+export const useSimpleTokenUser = <TUser extends Partial<IUser>>({
```

Comment:
just a suggestion, I know we have no time for that. But I think would be cool to move this `useSimpleTokenUser` into its own file `user/useSimpleTokenUser`, maybe we add it as a TODO?